Makefile

```diff
@@ -0,0 +1 @@
+nothing:
```

Comment:
are we sure that using nothing: aligns with our code style?

Reply:
Hm..... I think, it should be at least uppercased, but the better way is just cleanup that file, IMHO.

Reply:
@rakshazi Agree